lib/iex/resources/chart.rb

```diff
@@ -21,6 +21,8 @@ def self.get(symbol, range = nil, options = {})
         end
       rescue Faraday::ResourceNotFound => e
         raise IEX::Errors::SymbolNotFoundError.new(symbol, e.response[:body])
+      rescue Faraday::ClientError => e
+        raise IEX::Errors::BadRequestError, e.response[:body]['error']
```

Comment:
There's a bug here. You're passing in the text of the error, but the exception itself takes a `response` object. This should be `e.response` and the exception should be doing `super e.response[:body]['error']`.